index.d.ts

```diff
@@ -133,6 +133,38 @@ type ReaderRevenuePosition =
     | 'ampHeader'
     | 'ampFooter';
 
+interface CampaignsType {
+    id: string;
+    name: string;
+    rules: [];
+    priority: number;
+    activeFrom: number;
+    displayOnSensitive: boolean;
+    fields: {
+        formId: number;
+        callout: string;
+        _type: string;
```

Comment:
Is there a limited list of _types? I think it would be better to create strict types to start with and fix any errors by adding more types as they surface, rather than allowing everything